app/components/windows/AdvancedStatistics.m.less

```diff
@@ -0,0 +1,135 @@
+@import "../../styles/index.less";
+
+.container {
+  @media (max-width: 1200px) {
+    :global(.performance-metric-icon) {
+      display: inline-block !important;
+    }
+  }
+
+  :global(.performance-metric-wrapper) {
+    background-color: var(--section-alt);
+    padding: 10px;
+    margin-right: 2px;
+    flex-grow: 1;
+  }
+
+  :global(.performance-metric-wrapper)::before {
+    content: '' !important;
+    padding: 0;
+    padding-right: 1px;
+  }
+}
+
+.section {
+  margin-top: 40px;
+}
+
+.description {
+  margin-top: 20px;
+}
+
+.status {
+  padding: 10px 10px;
+  text-align: center;
+  margin-bottom: 20px;
+  .radius(1);
+
+  span {
+    margin-left: 5px;
+  }
+}
+
+.error {
+  background: rgba(251, 72, 76, 0.28);
```

Comment:
we should define this color in colors.less and assign it a variable if it doesn't already exist

Reply:
fixed